packages/paste-core/components/chat-log/stories/index.stories.tsx

```diff
@@ -12,6 +13,13 @@ export default {
   title: 'Components/ChatLog',
 };
 
+export const NewMessagesButton: StoryFn = () => (
```

Comment:
Suggestion(non-blocking): Ooooooooh ok this may be scope creep, but when I merged the chat log component, I added a new story called scrollable chat log. it would be awesome if instead of this story, you added in the new messages button to that story so people can see it in context.